itez/beneficiary/tests/test_utils.py

```diff
@@ -0,0 +1,39 @@
+from django.test import TestCase
+
+from itez.beneficiary.utils import generate_uuid_and_agent_code
+
+
+class UUIDAndAgentCode(TestCase):
+    @classmethod
+    def setUp(self):
+        """
+         This method is called for every test that runs and helps with variable declaration
```

Comment:
Break the Docstrings into 2 lines it's too long

Reply:
- [x] Docstrings breakdown